index.php

```diff
@@ -1,146 +1,383 @@
-<?php
+$(function () {
```

Comment:
You left off the opening `<?php` tag here, please be sure to test any changes you make to ensure that they work prior to pushing them up.